CMakeSettings.json

```diff
@@ -0,0 +1,30 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
```

Comment:
Is the folder `out` a standard location for build output?

Reply:
It's "standard" in the sense that it's the default location VS sets up for you